.github/workflows/build.yml

```diff
@@ -9,62 +9,37 @@ on:
   workflow_dispatch:
 
 jobs:
-  unix-build:
+  build:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-18.04, ubuntu-latest]
-        emacs: [26.1, 26.2, 26.3, 27.1, 27.2, snapshot]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Why is Windows here if it has its own step?

Reply:
This is what we are trying to do. It is having his own step using Cask, in Eask; they are the same! :)

Reply:
Ah right! Misread the diff.  Github isn't exactly mobile friendly :)